iis-common/src/main/java/eu/dnetlib/iis/common/java/io/HdfsUtils.java

```diff
@@ -1,37 +1,53 @@
 package eu.dnetlib.iis.common.java.io;
 
-import java.io.IOException;
-
 import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.fs.FileStatus;
 import org.apache.hadoop.fs.FileSystem;
 import org.apache.hadoop.fs.Path;
 
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
 /**
  * Util class containing operations on hdfs or local filesystem
- * 
+ *
  * @author madryk
  */
 public final class HdfsUtils {
 
     //------------------------ CONSTRUCTORS -------------------
-    
-    private HdfsUtils() {}
-    
+
+    private HdfsUtils() {
+    }
+
     //------------------------ LOGIC --------------------------
-    
+
     /**
      * Removes file or directory (recursively) located under the specified pathname.
      */
     public static void remove(Configuration hadoopConf, String pathname) throws IOException {
-        
         Path path = new Path(pathname);
-        
         FileSystem fileSystem = FileSystem.get(hadoopConf);
-        
```

Comment:
Including such unrelated formatting changes makes the review a bit harder than it would be otherwise.

Reply:
Yes, it also adds to pull request's changed lines value. It's an effect of applying IDE's code formatting for the entire file, not only for the code that's added.